modules/__tests__/keyframes-test.js

```diff
@@ -1,7 +1,8 @@
 var styleElement;
 var exenv;
 
-var {Component} = require('react');
+var React = require('react/addons');
```

Comment:
Ha, that's a great point actually... Got carried away from `resolve-styles` where we actually make use of `React`! Fix incoming any second now :)